site2/docs/tutorials-produce-consume.md

```diff
@@ -0,0 +1,62 @@
+---
+Id: tutorials-produce-consume
+title: Produce and consume messages
+sidebar_label: “Tutorials”
+—
+
+:::Note: This tutorial has the following prerequisites:
+[Create tenant](tutorials-tenant.md)
+[Create namespace](tutorials-namespace.md)
+[Create topic](tutorials-topic.md):::
+
+In this tutorial, we will:
+Configure the Pulsar client
+Create a subscription
+Create a producer
+Send test messages
+Verify the results
+
+In the ${PULSAR_HOME}/conf/client.conf file, replace webServiceUrl and brokerServiceUrl with your service URL.
+Create a subscription to consume messages from apache/pulsar/test-topic.
+bin/pulsar-client consume -s sub apache/pulsar/test-topic  -n 0

```

Comment:
```suggestion
```bash
bin/pulsar-client consume -s sub apache/pulsar/test-topic  -n 0
```
And it's required to add ``` as the last line for each command snippet. It applies to all occurrences.